build-tools/scripts/settings.gradle

```diff
@@ -15,25 +15,23 @@
  * limitations under the License.
  */
 
-includeBuild("${rootDir}/../geode-dependency-management")
```

Comment:
You’ve removed includeBuild for several build tools and now depend on devs first doing `publishToMavenLocal`. Consider using `includeBuild` with proper dependency substitution or implement a task dependency that ensures plugins are published before use.